restaurants/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+from django.http        import JsonResponse
+from django.views       import View
 
-# Create your views here.
+from users.models       import User
+from restaurants.models import Restaurant
+
+class RestaurantDetailView(View):
+    def get(self, request, restaurant_id):
+        try:
+            restaurant_instance = Restaurant.objects.get(id=restaurant_id)
+            fake_user_instance  = User.objects.get(id=1)
+            is_wished           = fake_user_instance.wishlist_restaurants.filter(id=restaurant_id).exists()
```

Comment:
`fake_user_instance`는 뭔가요?
```suggestion
            is_wished           = fake_user_instance.wishlist_restaurants.filter(id=restaurant_id).exists()
```

Reply:
아직 decorator (token과 일치하는지 확인하는 로직)을 merge하지 않은 상태여서 저렇게 했었습니다! 
이 branch에서는 에러가 나더라도 decorator을 호출하면 될까요?

Reply:
merge됐을 때 생각해서 토큰 받는 decorator 호출하는 쪽으로 했습니다!

Reply:
이런 경우는 decorator코드가 merge되고 나서 올리는게 맞습니다!
main branch에 merge 되는 코드는 제대로 동작하는 코드여야 합니다~

decorator PR 머지되었으니 마지막 테스트 해보시고 코멘트 남겨주세요~~